appserver/webservices/webservices-scripts/src/main/resources/glassfish/bin/schemagen.bat

```diff
@@ -56,4 +57,10 @@ goto run
 set JAVA=java
 
 :run
-%JAVA% -Djava.endorsed.dirs="%~dp0..\modules\endorsed" -cp "%~dp0..\modules\webservices-osgi.jar;%~dp0..\modules\javax.xml.rpc-api.jar;%~dp0..\modules\jaxb-osgi.jar" com.sun.tools.jxc.SchemaGeneratorFacade %*
+CALL jdkcheck.bat

```

Comment:
Same like for bash scripts, you need to use `%~dp0` to specify path, these are seldom called directly:
`CALL %~dp0jdkcheck.bat `